src/olympia/blocklist/tasks.py

```diff
@@ -48,6 +48,13 @@ def BLOCKLIST_RECORD_MLBF_BASE(block_type: BlockType):
             raise ValueError(f'Unknown block type: {block_type}')
 
 
+@task
+def upload_mlbf_to_remote_settings(force_base=False):
```

Comment:
So this is confusing because the cron has the same name, but doesn't use this task. I don't think we want to change the cron at this time, so maybe rename the task to `upload_mlbf_to_remote_settings_trigger` or `upload_mlbf_to_remote_settings_admin` or something, and add a short comment explaining why it's there. Then you don't need to do the `as` bit in the import below as it wouldn't conflict anymore.

Reply:
Done.